src/cmd/config.go

```diff
@@ -70,16 +71,19 @@ func init() {
 }
 
 func LoadConfig() (*common.Config, error) {
-	v := &common.ConfigVersion{}
```

Comment:
The `type ConfigVersion struct` in config.go should also be deleted since it's unused now.